scripts.d/s-salome

```diff
@@ -591,7 +591,9 @@ function s-salome-wrapup()
 
                 # --sha256 in 2.4 only
 
-                if [[ $(is-enabled xz) ]]; then
+                if [[ $SCBI_IS_WINDOWS == yes ]]; then
+                    O_COMPRESS="-9r"
```

Comment:
`-9r` is not a compression format. 9 is for max compression and r for recursive. I think you should remove this and...